content/intro-to-storybook/react/en/composite-component.md

```diff
@@ -81,29 +72,58 @@ export const defaultTasksData = [
   { ...taskData, id: '5', title: 'Task 5' },
   { ...taskData, id: '6', title: 'Task 6' },
 ];
-
 export const withPinnedTasksData = [
   ...defaultTasksData.slice(0, 5),
   { id: '6', title: 'Task 6 (pinned)', state: 'TASK_PINNED' },
 ];
```

Comment:
I think we should drop these exports and instead just reuse the args.

Reply:
@jonniebigodes maybe we can figure out the jest thing in Slack? It sounds familiar.

In your test you should be able to use `Default.args.tasks` just fine!

Reply:
@tmeasday i've updated the test in the composite chapter. The initial issue that i had with jest and fsevents is now fixed. After a bit of research it seems that deleting both `yarn.lock` and `node_modules` and then issue `yarn` to install the dependencies works in some cases, it did for me, but can't promise that this is a silver bullet (pardon the bad pun), probably something might be wrong with the cra templated that is installed. 

One item that needs a bit of attention, it seems that the actions are not propagated, running the tests now yield a warning mentioning undefined but they are marked as required. Doing a simple test i was able to bypass it through removing the required from the proptype. It didn't have impact on the snapshots and the actual tests that we're mentioning. 

Let me know if you're ok by changing this?